CERTIFICATION.md

```diff
@@ -0,0 +1,18 @@
+## Accelerator Program Certification Process.
+The following guidelines will be used in certifying the CHT Accelerator program grantees. App Developers and grantee organizations will be certified after completing the CHT app building course and also demonstrating that after the CHT onboarding the organizations are able to design and build CHT prototypes or workflows. 
+
+### 1. CHT app building course certification
+App Developers from participating technical organizations who complete the CHT app building course will be issued with the CHT app building academy course completion certificates.
+
+### 2. Certification of grantee organizations.
+The Accelerator organizations will be certified if they demonstrate that after being onboarded on CHT, the organizations are able to design and build CHT workflows and use cases
+
+#### Evaluation criteria 
+The design process and outputs should be well documented
+- The user requirements, technical design guides and prototypes should be well documented.
+  
+**Recommended configuration best practices and configuration outputs**
+- The code should follow the recommended [coding style](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/).
```

Comment:
```suggestion
- The code should follow the recommended [coding style](https://github.com/medic/cht-accelerator#readme).
```

Reply:
@antonykhaemba Can we use the readme file instead?It is more detailed.

Reply:
Yes we can use that.